Calculator.Engine/Calculator.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Calculator.Engine
+{
+    public class Calculator
+    {
+        public int Add(int firstOperator, int secondOperator)
```

Comment:
operator is not a proper name, because it means something else ... What you meant is "operand"